tests/screenshot/apps/components/LabeledIcon.js

```diff
@@ -2,8 +2,10 @@ import LabeledIcon from '../../../../LabeledIcon';
 
 const LabeledIconTests = [
 	<LabeledIcon />,
-	<LabeledIcon flip="horizontal" />,
-	<LabeledIcon icon="time" />
+	<LabeledIcon flip="horizontal" icon="temperature">Hello LabeledIcon</LabeledIcon>,
+	<LabeledIcon icon="time">Hello LabeledIcon</LabeledIcon>,
+	<LabeledIcon icon="menu" labelPosition="left">Hello LabeledIcon</LabeledIcon>,
+	<LabeledIcon icon="search" size="small">Hello LabeledIcon</LabeledIcon>
```

Comment:
Can you add more ss-tests?  examples : all sizes; all label positions; inline; disabled